starter-code/js/index.js

```diff
@@ -1,52 +1,76 @@
-function deleteItem(e){
+// ITERATION 1 OLD
+// var calculatePriceBtn = document.querySelector(".btn-success");
+// var priceInput = document.querySelector("input");
+// var unitPrice = document.querySelector(".one-unit-cost");
+// var priceResult = document.querySelector(".result");
 
-}
+// calculatePriceBtn.onclick = function(){
+// var result = parseFloat(unitPrice.innerHTML.slice(1)) * parseFloat(priceInput.value);
+// priceResult.innerHTML = "$" + result;
+// };
 
-function getPriceByProduct(itemNode){
+//ITERATION 2 OLD
+// var calculatePriceBtn = document.querySelector(".btn-success");
 
-}
+// var productOne = document.querySelector(".product-one");
+// var priceInput = productOne.querySelector("input");
+// var unitPrice = productOne.querySelector(".one-unit-cost");
+// var priceResult = productOne.querySelector(".result");
 
-function updatePriceByProduct(productPrice, index){
+// var productTwo = document.querySelector(".product-two");
+// var priceInputTwo = productTwo.querySelector("input");
+// var unitPriceTwo = productTwo.querySelector(".one-unit-cost");
+// var priceResultTwo = productTwo.querySelector(".result");
 
-}
+// calculatePriceBtn.onclick = function(){
+// var result = parseFloat(unitPrice.innerHTML.slice(1)) * parseFloat(priceInput.value);
+// priceResult.innerHTML = "$" + result;
+// var resultTwo = parseFloat(unitPriceTwo.innerHTML.slice(1)) * parseFloat(priceInputTwo.value);
+// priceResultTwo.innerHTML = "$" + resultTwo;
 
-function getTotalPrice() {
 
-}
+// var finalResult = document.querySelector(".final-price");
+// finalResult.innerHTML = result + resultTwo;
 
-function createQuantityInput(){
+// };
 
-}
 
-function createDeleteButton(){
+//ITERATION 1, 2 AND 3
 
-}
 
-function createQuantityNode(){
+var calculatePriceBtn = document.querySelector(".btn-success");
+var allProducts = document.querySelectorAll(".product");
```

Comment:
Here you define `allProducts` as all DOM elements that have the "product" class. Which is your two products, but also the product input, and this third unwanted element makes your `totalPrice()` function go nuts :)